styles/index.css

```diff
@@ -52,3 +116,83 @@
   font-family: var(--heading-sans);
   font-weight: 600;
 }
+.left-corner p {
+  margin-left: 0;
+}
+.blogs-section-container {
+  padding: 30px 88px;
+  font-family: var(--paragraph-sans);
+}
+.blogs-header-container {
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+}
+.blogs-header-container h1,
+p {
+  margin: 10px;
+}
+.blogs-header-container h1 {
+  padding: 0;
+}
+.blogs-carousel {
+  display: flex;
+  justify-content: flex-end;
+}
+
+.left-button,
+.right-button {
+  width: 15px;
+  height: 15px;
+  border: none;
+  background-color: inherit;
+  cursor: pointer;
+  padding: 0;
+  margin: 0;
+}
+.carousel-buttons {
+  display: flex;
+  gap: 10px;
+  margin-bottom: 10px;
+}
+.arrow-icon {
+  width: 100%;
+}
+
+.banner-image {
+  width: 225px;
+  height: 155px;
```

Comment:
try to specify a size the perserves image assets ratio

Reply:
.banner-image {
  width: auto;
  height: 150px;
} so I fixed the height while the width to be auto. However, the 2 last images still not have the same width as maybe their assets ratio is not the same as the remaining. Is that OK or all the images should have the same width and height while keeping their assets ratio.

Reply:
this solution is invalid for now as I needed to set width:100% to align the arrow and the content. Will get back to this tomorrow.

Reply:
Thanks for the clarification. 
the images look much better now